.github/actions/spell-check/expect.txt

```diff
@@ -1,6 +1,3 @@
-# FALSE POSITIVES
-## "PackagemanagerWrapper.cs" should be "PackageManagerWrapper.cs"
-## NOTICE.MD > MOZILLA PUBLIC LICENSE v1.1
```

Comment:
Unfortunately, check-spelling v0.0.24 doesn't like comments. That's actually a regression -- there's code to support it, but there were also cases where people incorrectly added words like `PackagemanagerWrapper` expecting a certain behavior that doesn't match how check-spelling behaves, so there's now a warning about that.

I'll include a fix to restore support for comments as well as adding a test to ensure that I don't break it going forward...

Reply:
To be honest, these comments just end up being pushed up after people run the local "add to expect" script 😆 They end up not mattering much anymore since they're no longer close to the entries they were explaining. Those have been sorted as well.

Reply:
Yeah, I think that was my argument against adding support for comments -- I don't really have a way for that script to handle them -- it would require it to split the file into segments (delimited by `#`) and sort each segment. It isn't impossible, but it's considerably more work -- whereas people could just use multiple expect files and have notes about why the files exist.